src/docs/style/illustrations.mdx

```diff
@@ -0,0 +1,81 @@
+
+You should adhere to these guidelines when creating your own illustrations. Alternatively, you may also use existing illustrations/artwork from repositories such as [Icons8](https://icons8.com/illustrations), [Adobe Stock](https://stock.adobe.com/vectors), or [Humaaans Figma plugin](https://www.figma.com/community/plugin/739503328703046360). We recommend finalizing your illustrations with [Adobe Illustrator](https://www.adobe.com/products/illustrator.html), as it provides more export options than other tools.
+
+<React.Fragment>
+    <MaterialDesignDescription
+        title={'Onboarding'}
+        url={'https://material.io/design/communication/onboarding.html'}
+        description={`Material Design's guidelines on onboarding screens.`}
+    />
+    <MaterialDesignDescription
+        title={'Google Marketing Platform Visual Guidelines'}
+        url={
+            'https://designguidelines.withgoogle.com/ads-branding/assets/1hquKP2EcBlDaWyaLkoezqd8r9KPjRgnp/visual-guidelines-google-marketing-platform.pdf#'
+        }
+        description={`You are encouraged to follow their illustration guidelines (page 42 - 48).`}
+        avatar={<PDFIcon style={{ width: 40, height: 40 }} />}
+    />
+</React.Fragment>
+
+## Designing the Illustration
+
+### Perspective
+
+Your illustration should be either facing the subject matter parallel to the ground (not viewing it from above or below) or in parallel projection.
+
+### Ratio
+
+The aspect ratio of your illustrations should be between 1:1 and 2:3 to avoid filling the full screen on mobile devices (leave room for buttons, steppers, etc.). If you are designing exclusively for desktop applications (i.e., the illustration will never show up on mobile), you can go wider, but no more than 16:9.
+
+### Drawing Style
+
+Keep your style consistent within the application. This style should also extend to any marketing sites for the product for consistency. Do not be too abstract or artistic — your main goal is to inform your users of the features in your product, not to decorate it.
+
+### Text
+
+Avoid the use of text wherever possible. It can be hard to see on mobile and will be ineffective if your application needs to support multiple languages. If you have to use text, use Open Sans as the font and ensure that words are legible on your smallest targeted device.
+
+### Color
+
+Use the [PX Blue color palette](/style/color) for your [local colors](<https://en.wikipedia.org/wiki/Local_color_(visual_art)>). You should use some shade of blue in the foreground of your images (these can be colors from the Blue, Light Blue, or Branding Blue palettes).
+
+Avoid using colors that may blend into the app theme background (suggest adding a background color to pad around it). Avoid using the 500 colors from the from the User Interface palettes, as they may blend with other UI elements (buttons, progressors).
```

Comment:
This paragraph is the old version of the one below it...i think we had a merge snafu
```suggestion
```

Reply:
Thanks for checking :)